lib/jekyll-github-metadata/repository.rb

```diff
@@ -62,6 +62,64 @@ def repo_pages_info_opts
         end
       end
 
+      # Whitelisted keys for Organizations and Users
+      WHITELISTED_ORGANIZATION_KEYS = Set.new([
+        :login,
+        :id,
+        :node_id,
+        :url,
+        :avatar_url,
+        :description,
+        :name,
+        :company,
+        :blog,
+        :location,
+        :email,
+        :is_verified,
+        :has_organization_projects,
+        :has_repository_projects,
+        :public_repos,
+        :public_gists,
+        :followers,
+        :following,
+        :html_url,
+        :created_at,
+        :type,
+        :collaborators,
+      ])
+
+      WHITELISTED_USER_KEYS = Set.new([
+        :login,
+        :id,
+        :node_id,
+        :avatar_url,
+        :html_url,
+        :type,
+        :site_admin,
+        :name,
+        :company,
+        :blog,
+        :location,
+        :bio,
+        :public_repos,
+        :public_gists,
+        :followers,
+        :following,
+        :created_at,
+        :updated_at,
+      ])
+
+      def owner_obj
```

Comment:
Instead of owner_obj, how about calling this `owner_metadata`? A comment above the method would also help tremendously! I'm also tempted to move this user lookup & filter into a separate Drop class...

Reply:
Done.

(When first writing the code, I was thinking of a follow up that would rename `owner` to `owner_name` to make those match... It's a much more intrusive patch, but maybe it would be nice to have the same name on either side?)

In any case, renamed as requested. Please take another look.